be/src/io/shared_buffered_input_stream.cpp

```diff
@@ -20,7 +20,7 @@ namespace starrocks::io {
 
 SharedBufferedInputStream::SharedBufferedInputStream(std::shared_ptr<SeekableInputStream> stream,
                                                      const std::string& filename, size_t file_size)
```

Comment:
if it's reference, then no need to use move.

Reply:
otherwise reference data will be clean.

Reply:
fixed

Reply:
actually `std::move()` can't change const reference data